keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -15,6 +16,7 @@ sealed class ClusterDeployStrategy {
 }
 
 data class RedBlack(
+  override val considerOnlyAmazonHealth: Boolean = false,
```

Comment:
My suggestion is `considerOnlyInstanceHealth`, to make it cloud-provider-agnostic.